code/modules/antagonists/separatist/separatist.dm

```diff
@@ -73,12 +76,21 @@
 	show_in_antagpanel = FALSE
 	show_name_in_check_antagonists = TRUE
 	suicide_cry = "FOR THE MOTHERLAND!!"
+	///team datum
 	var/datum/team/nation/nation
 
 /datum/antagonist/separatist/on_gain()
 	create_objectives()
 	. = ..()
 
+//give ais their role as UN
+/datum/antagonist/separatist/apply_innate_effects(mob/living/mob_override)
+	. = ..()
+	if(isAI(mob_override))
+		var/mob/living/silicon/ai/united_nations_ai = mob_override
+		united_nations_ai.laws = new /datum/ai_laws/united_nations
+		united_nations_ai.laws.associate(united_nations_ai)
```

Comment:
unsynced borgs too?

Reply:
they're unsinked so meh. science, for example, it makes sense for them to unslave borgs they create.